CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ UNRELEASED
 * [ [#1350](https://github.com/digitalfabrik/integreat-cms/issues/1350) ] Various small PDF export improvements
 * [ [#1777](https://github.com/digitalfabrik/integreat-cms/issues/1777) ] Fix autocompleting POI address for non-staff users
 * [ [#1749](https://github.com/digitalfabrik/integreat-cms/issues/1749) ] Fix region deletion error if media library has nested structure
+* [ [#1762](https://github.com/digitalfabrik/integreat-cms/issues/1762) ] Disable upload-test-coverage CircleCI job for PRs of forks
```

Comment:
```suggestion
```
Since this isn't a functional change to the CMS, I would not put it into the changelog... Our user's wont see a difference by this change.

Reply:
ah, ok. sure